src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-pymssql.py

```diff
@@ -10,4 +10,10 @@
 # SPDX-License-Identifier: GPL-2.0-or-later
 # ------------------------------------------------------------------
 
-hiddenimports = ['_mssql']
+from PyInstaller.utils.hooks import is_module_satisfies
+
+hiddenimports = ["_mssql"]
```

Comment:
This should probably go into else branch, because it is available as top-level module in those older versions.